docs/modules/plugins/pages/plugin-azure-resource-manager.adoc

```diff
@@ -71,6 +71,25 @@ resourceGroups/sample-resource-group/providers/Microsoft.KeyVault/vaults/sample-
 When I get Azure resource with identifier `resourceGroups/TEST-RG/providers/Microsoft.KeyVault/vaults/KEY-VAULT-NAME` using API version `2021-10-01` and save it to scenario variable `key-vault-properties`
 ----
 
+==== Get information about Azure resource using Azure resource URL
+
+Gets the info about the specified Azure resource using the declared Azure resource URL and saves it to a variable. For more information, see the
+https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API reference].
+
+[source,gherkin]
+----
+When I get Azure resource with URL `$azureResourceUrl` and save it to $scopes variable `$variableName`
+----
+
+* `$azureResourceURL` - It's used to specify Azure resource uniquely. For example:
++
+[source,gherkin]
+----
+https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-resource-group/providers/Microsoft.KeyVault/vaults/sample-vault?api-version=2021-10-01
```

Comment:
please add more examples, but with another domain, not `management.azure.com`

Reply:
Added Log Analytics URL example, added example of usage with Key Vault.